Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -1061,45 +1058,47 @@ private void updateActionsForLargeLandscape(Menu menu) {
         menu.findItem(R.id.menu_empty_trash).setVisible(mSelectedTag != null && mSelectedTag.id == TRASH_ID);
     }
 
-    public void updateViewsAfterTrashAction(Note note) {
+    public void trashNote(Note note) {
         if (note == null || isFinishing()) {
             return;
         }
 
+        note.setDeleted(!note.isDeleted());
+        note.setModificationDate(Calendar.getInstance());
+        note.save();
+
         if (note.isDeleted()) {
             List<String> deletedNoteIds = new ArrayList<>();
             deletedNoteIds.add(note.getSimperiumKey());
             mUndoBarController.setDeletedNoteIds(deletedNoteIds);
             mUndoBarController.showUndoBar(getUndoView(), getString(R.string.note_deleted));
             AnalyticsTracker.track(
-                LIST_NOTE_DELETED,
-                CATEGORY_NOTE,
-                "overflow_menu"
+                    LIST_NOTE_DELETED,
+                    CATEGORY_NOTE,
+                    "overflow_menu"
             );
         } else {
             AnalyticsTracker.track(
-                EDITOR_NOTE_RESTORED,
-                CATEGORY_NOTE,
-                "overflow_menu"
+                    EDITOR_NOTE_RESTORED,
+                    CATEGORY_NOTE,
+                    "overflow_menu"
```

Comment:
```suggestion
                EDITOR_NOTE_RESTORED,
                CATEGORY_NOTE,
                "overflow_menu"
```

This might have been automatically edited by Android Studio.  The edit and extra indention is not necessary.

Reply:
If we want to avoid those, we need to clarify that the code style to use (preferably in the contributing guide), and why not add a checkstyle file like FluxC 🙂.
For the current PR, I created a codestyle config from the checkstyle file in our FluxC repo, and removed the all the wrapping options (as they will create a lot changes), then reformatted the files: NotesActivity.java, NoteListFragment.java and NoteUtils.java e1fa30d2d524d614e51951f42fd32721b1dc87eb, please let me know what you think?

Reply:
There is a ton of legacy code with various styles due to different maintainers over the years.  While code style consistency is good in general, there are too many inconsistencies in the codebase to enforce something like checkstyle without having alert fatigue constantly and modifying code that is unrelated to the changes for the bug/enhancement/etc.  Plus, there are always exceptions to the rules.